### DISCUSSION TITLE
Recognoize USB-Type C port

### DISCUSSION THREAD

**Discussion by CanasCrespas** [dat-linux/community#63] [Issues]
Hi, I have an VivoBook_ASUSLaptop with datlinux, and I'm trying to share my screen on two monitors. The computer only has one HDMI port, and when I connect it, one monitor works fine. However, when I connect it to an HDMI to USB Type-C converter, the computer doesn't recognize it. I've used this converter with a Windows PC, and it works fine. Can you help me troubleshoot this?. Do you have any solution to this? maybe an script or a driver? thanks

**Comment by dat-linux** [comment]
Your best bet is trying a general Ubuntu forum. Find out the brand and make of the adapter, others may have a similar issue. DAT Linux is based on Ubuntu 22.04.
